CONTRIBUTING.md

```diff
@@ -0,0 +1,31 @@
+# Contributing to USWDS web components
+
+Thank you for your interest in contributing to USWDS web components.
+
+> [!Important]
+>
+> **This repo is currently in beta and is not yet ready for community code contributions.**
+>
+> While the USWDS core team works through big decisions about architecture and style, we ask that you do not submit pull requests. Instead, we encourage you to contribute by your sharing your ideas, concerns, questions, and experiences with the USWDS core team and community.
+>
+> Learn more about how you can provide feedback in the [submitting an issue](#submitting-an-issue) and [starting a discussion](#starting-a-discussion) sections below.
+
+## How can I contribute?
+
+### Submitting an issue
+
+If you have discovered a bug, let us know by [submitting an issue on GitHub](https://docs.github.com/en/issues/tracking-your-work-with-issues/creating-an-issue). We’ll address it as soon as we are able (but because we are in beta, please be aware that it might be awhile).
+
+### Starting a discussion
+
+If you want to share ideas, questions, or concerns with the USWDS core team and community, [open a discussion on GitHub](https://docs.github.com/en/discussions/quickstart#creating-a-new-discussion).
+
+### Submitting a pull request
+
+At this time, we are not accepting code contributions in the community. We expect to open up community pull request submissions in early 2025, so check back later for instructions on how to contribute pull requests to USWDS web components.
```

Comment:
**suggestion:** Emphasis could be useful for scannability.

```suggestion
At this time, we are not accepting <u>code</u> contributions in the community. 

We expect to open up community pull request submissions in early 2025. Check back later for instructions on how to contribute code.
```

Reply:
I did not include this because the `<u>` element doesn't appear to do anything in the markdown. Additionally, after learning more about the `<u>` element, I lean away from using it because it's styling might cause some users to think it is a link. We could use italics or bold to emphasize the text, but that felt like it created a different tone as I was reading it. Open to thoughts here!